packages/zoe/src/zoe.js

```diff
@@ -321,7 +321,7 @@ const makeZoe = (additionalEndowments = {}) => {
         }
       }
       const installationHandle = installationTable.create(
-        harden({ installation }),
+        harden({ installation, code }),
```

Comment:
What's the difference between the installation and the code?

Reply:
`code` is a string with the actual source code. `installation` is the result of evaluating the code with all the context it needs. It's an object with a `makeContract()` function. That function has a bunch of metering wrappers thrown in, so it's not a useful place for clients to get access to and compare the code they are expecting to see.